Cosmostation/Controller/Main/CommonWCViewController.swift

```diff
@@ -1299,4 +1299,18 @@ extension CommonWCViewController {
             }
         }
     }
+    
+    @MainActor
```

Comment:
:)
I am curious about the threading strategy, I have seen DispatchQueue everywhere and is the first time I see a @MainActor. 
Have you got any plan/guideline to use one or another? :)

Reply:
soory but not have concrete plane
and now we integrate wallet connect v1 and v2 together.
It makes some redundant works